codes.json

```diff
@@ -505,5 +525,15 @@
       "doc": "Official Documentation @ https://datatracker.ietf.org/doc/html/rfc7540#section-9.1.2",
       "description": "Defined in the specification of HTTP/2 to indicate that a server is not able to produce a response for the combination of scheme and authority that are included in the request URI."
     }
+  },
```

Comment:
It looks like HTTP 521 is not an official status code, but instead is specific to CloudFlare. The documentation linked in this PR [here](https://datatracker.ietf.org/doc/html/rfc7504#section-3) is for an unrelated SMTP (not HTTP) error.

@angellee177 or @angelsupp remove this status code ( 521 ) and this PR will be merged as suggested by @prettymuchbryce